pre-processing/primod/ribamod/exchange_creator.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+from numpy.typing import NDArray
+from scipy.spatial import KDTree
+
+Int = np.int_
+Float = np.float_
+Bool = np.bool_
```

Comment:
If we start following that convention, let's put those constants somewhere else and change all instances of `np.int_`, ... in primod

Reply:
What I use in a number of other libraries is things like `FloatArray, IntArray, BoolArray`, but mostly out of necessity due to older numpy versions.

I think NDArray[Int] looks just as good though, and is probably clearer?

Reply:
I'll add a typing module, we can expand its use later. Because I think it'll potentially generate some merge conflicts for Robert and Hendrik as well.

Reply:
I don't think they modify the pre-processor much at the moment, but up to you